applications/crossbar/src/modules/cb_channels.erl

```diff
@@ -508,3 +511,10 @@ maybe_intercept(Context, CallId, TargetType, TargetId) ->
     lager:debug("attempting to move ~s to ~s(~s)", [CallId, TargetId, TargetType]),
     kz_amqp_worker:cast(API, fun kapi_metaflow:publish_req/1),
     crossbar_util:response_202(<<"intercept initiated">>, Context).
+
+-spec get_account_id(cb_context:context()) -> ne_binary().
+get_account_id(Context) ->
+    case cb_context:account_id(Context) of
+        'undefined' -> <<"all">>;
+        AccountId -> AccountId
+    end.
```

Comment:
@onnet please add a system config to enable this search, should default to `false`. when you potentially have thousands of channels you don't want admins to inadvertently use this option